### PR TITLE
fixing pathway of Transaction Fees

### PR DESCRIPTION
**Pull request by Budalebah**
correction of pathway https://docs.optimism.io/stack/transactions/fees

### CODE REVIEW

File: pages/tokens.mdx [hunk 1]
```diff
@@ -1,26 +1,14 @@
...
 - Request must be created by a representative of the project or include a verifying tweet from the project Twitter account.
 
-<Cards>
-  <Card
-    icon={<PiArrowSquareInDuotone size={32} />}
-    title="Token Request Form"
-    href="https://superseedxyz.typeform.com/to/c2J8j8m7"
-  />
-</Cards>
+[Token Request Form](https://superseedxyz.typeform.com/to/c2J8j8m7)
```
Comment (0xNikolas) [inline pages/tokens.mdx:14]: What is the reason for removing the card?

**Review by 0xNikolas — CHANGES_REQUESTED** [review]

File: pages/tokens.mdx [hunk 2]
```diff
@@ -1,26 +1,14 @@
...
-
 # Superseed Token List
 
 ## Testnet
 
-<Callout type="info" emoji="ℹ️" title="Guidelines">
-  To list your token on the Superseed testnet, use this form and ensure your
-  token follows the guidelines below:
-</Callout>
+ℹ️ **Guidelines**
```
Comment (0xNikolas) [inline pages/tokens.mdx:5]: Any reason you removed the Callout?

**Review by 0xNikolas — CHANGES_REQUESTED** [review]

**Comment by 0xNikolas** [comment]
Please create a pull request with only this changed file `pages/build-on-superseed/differences-compared-to-ethereum.mdx`

**Comment by Budalebah** [comment]
https://github.com/superseed-xyz/docs/pull/50/commits/a6fdf371519724ddc40ee5ff719b14f3f2041037 done

**Pull request closed without merging**